src/bindgen/ir/function.rs

```diff
@@ -205,6 +213,10 @@ impl Source for Function {
                 write!(out, " {}({})", swift_name_macro, func.swift_name());
             }
 
+            if let Some(no_return_attr) = &config.function.no_return {
+                out.write_fmt(format_args!(" {} ", no_return_attr));
```

Comment:
This needs to somehow be conditional on `never_return`, right? Please add a non-never-returning function to the new test.